xonsh/xoreutils/uname.py

```diff
@@ -0,0 +1,209 @@
+import platform
+import sys
+from xonsh.xoreutils.util import arg_handler
```

Comment:
please add a single blank line between standard library imports and package imports